modules/test/playwright/tests/export-import-web/import.spec.ts

```diff
@@ -479,3 +483,105 @@ test('can import custom object entries at instance level with or without permiss
 		})
 	);
 });
+
+test('Can/not view Import menu item in Application menu depending on permissions', async ({
+	apiHelpers,
+	applicationsMenuPage,
+	page,
+}) => {
+	const companyId = await page.evaluate(() => {
+		return Liferay.ThemeDisplay.getCompanyId();
+	});
+
+	const user1 = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[user1.alternateName] = {
+		name: user1.givenName,
+		password: 'test',
+		surname: user1.familyName,
+	};
+
+	const roleWithPermissions = await apiHelpers.headlessAdminUser.postRole({
+		name: 'role' + getRandomInt(),
+		rolePermissions: [
+			{
+				actionIds: ['VIEW_CONTROL_PANEL'],
+				primaryKey: companyId,
+				resourceName: '90',
+				scope: 1,
+			},
+			{
+				actionIds: ['ACCESS_IN_CONTROL_PANEL'],
+				primaryKey: companyId,
+				resourceName:
+					'com_liferay_exportimport_web_portlet_CompanyImportPortlet',
+				scope: 1,
+			},
+		],
+	});
+
+	await apiHelpers.headlessAdminUser.assignUserToRole(
+		roleWithPermissions.externalReferenceCode,
+		user1.id
+	);
+
+	const user2 = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[user2.alternateName] = {
+		name: user2.givenName,
+		password: 'test',
+		surname: user2.familyName,
+	};
+
+	const roleWithoutPermissions = await apiHelpers.headlessAdminUser.postRole({
+		name: 'role' + getRandomInt(),
+		rolePermissions: [
+			{
+				actionIds: ['VIEW_CONTROL_PANEL'],
+				primaryKey: companyId,
+				resourceName: '90',
+				scope: 1,
+			},
+		],
+	});
+
+	await apiHelpers.headlessAdminUser.assignUserToRole(
+		roleWithoutPermissions.externalReferenceCode,
+		user2.id
+	);
+
+	await performLogout(page);
+
+	await performLogin(page, user1.alternateName);
+
+	await applicationsMenuPage.goToApplicationsMenu();
+
+	const importButton = page.getByRole('menuitem', {
```

Comment:
Can we use the existing buttons inside the ExportPage? Not sure if this button already exists there